aip/0122.md

```diff
@@ -24,11 +24,11 @@ defined by an API **must** be unique within that API.
 Resource names are formatted according to the [URI path schema][], but without
 the leading slash:
 
-    shelves/123/books/les-miserables
+    publishers/123/books/les-miserables
```

Comment:
This is probably getting super nitpicky, but publishers have natural names whereas shelves didn't, so it made sense for shelf IDs to be numerical, but it seems odd for publisher IDs to be numerical.

Reply:
I think it's reasonable for publisher IDs to be autoassigned, personally.

Reply:
I think either one is reasonable (and in fact this AIP shows both).